templates/aws-stack.yml

```diff
@@ -103,12 +105,6 @@ Parameters:
       - edge
     Default: "stable"
 
-  BuildkiteAgentToken:
-    Description: Buildkite agent registration token
-    Type: String
-    NoEcho: true
-    Default: ""
```

Comment:
If we're removing a parameter, I think we'll need to release it as v6.x. If we're doing **that** there's a list of possible breaking changes we've been holding off over in https://github.com/buildkite/elastic-ci-stack-for-aws/issues/792

Reply:
This must be a merge artifact because the parameter is still present just lower, I can fix this up in a changeset